Herbaria-Project-Outline.md

```diff
@@ -0,0 +1,97 @@
+
+1. Increase evaluation dataset to 1000 images
+2. Improve transcription accuracy of the taxon label
+3. Extract and transcribe the following for minimal metadata capture:
+    a. Geography 
+    b. Collection code
+    c. Barcode
+4. For detailed data capture:
+    a. Date Collected
+    b. Collector Name
+    c. Collector Number
+    d. Habitat
+5. Produce clean code and documentation
+
+
+### C. Provide a solution in terms of human actions to confirm if the task is within the scope of automation through AI.
+
+1. OCR and Transcription Improvement:
+
+A person would look at the plant image embedded with text, aiming to transcribe this text. Their primary focus is on the taxon label, and the surrounding textual content within the image. In addition, they would be on the lookout for specific items such as geography, collection code, barcode, date collected, collector name, collector number, and habitat.
+
+To ensure accuracy, this person would cross-reference the taxon label with a known list, ensuring it's either a valid label or closely matching one. If they encounter text that's challenging to interpret, they would draw from adjacent text or other contextual cues within the image to infer the likely content. This detailed transcription and validation task aligns well with capabilities of state-of-the-art OCR models, especially with refinements to prioritize the extraction of these newly outlined items.
+
+2. Plant Classification:
+
+Once textual content is identified or masked, the person would then shift their focus to the visual aspects of the plant image. Drawing from their prior knowledge of known plant categories, as well as potential hints from the transcribed information like the taxon label, geography, and collection code, they would determine the plant's identity. Recognizing a plant based on its visual characteristics and aligning it with external clues remains a task achievable with AI, particularly when using advanced computer vision methods and a robust training dataset.
+
+3. Plant Features Identification:
+
+Upon examining the plant image, the person would mark out specific features like fruits and flowers, noting their locations. They would count the number of such features and also categorize them based on their phenological stage, i.e., the stage in their life cycle. This identification and classification step pertains to AI's capabilities in image segmentation and classification, given a well-structured dataset. in.
+
+
+### D. Outline a path to operationalization.
+
+The proposed workflow for digitizing herbarium specimen is as follows:
+
+- Imaging: A customized photo station for imaging herbarium sheets and an automated image processing pipeline is used for digitizing the sheets.
+- Label transcription: Extracting the label from the specimen sheet, transcribing the label:
+-- Minimal metadata capture 
+-- Detailed data capture
+- Georeferencing: generation of latitude, longitude, and associated uncertainties.
```

Comment:
I don't think they are asking for lat/long, just the location written in the label.